.github/workflows/build.yaml

```diff
@@ -532,7 +532,7 @@ jobs:
       - name: Install Linux build dependencies
         if: ${{ runner.os == 'Linux' }}
         run: |
-          sudo apt update; sudo apt install -y python3-qpid-proton libpython3-dev ninja-build
+          sudo apt update; sudo apt install -y python3-qpid-proton libpython3-dev ninja-build libwebsockets-dev libnghttp2-dev
```

Comment:
this is a disadvantage of the change; cmake creates some `py` files (from `py.in`) files and there is now no way to have that part of the build run without providing all dependencies for the rest of the build